python/ml4ir/base/data/relevance_dataset.py

```diff
@@ -65,7 +65,7 @@ def create_dataset(self, parse_tfrecord=True):
         elif self.data_format == DataFormatKey.TFRECORD:
             data_reader = tfrecord_reader
         else:
-            raise NotImplementedError
+            raise NotImplementedError("Unsupported data format: {}".format(self.data_format))
```

Comment:
Looks good to me, but if you do another pass, please add ("Unsupported data format: {}. We currenty support {} and {}.".format(self.data_format, DataFormatKey.CSV, DataFormatKey. TFRECORD)) so that a user know what to do